spec/mailers/access_granted_mailer_spec.rb

```diff
@@ -1,15 +1,32 @@
 require 'spec_helper'
 
 describe AccessGrantedMailer do
-  before { create_magic_assessments }
-  let(:subject){ AccessGrantedMailer }
-  let(:assessment){ @assessment_with_participants }
-  let(:user){ Application.create_user } 
-  let(:role){ "facilitator" }
-
-  it "#notify" do
-    mail = subject.notify(assessment, user, role)
-    expect(mail.to).to include(user.email)
-    expect(mail.body).to include(role)
+  let(:subject) {
+    AccessGrantedMailer
+
```

Comment:
That's something we should tackle with a tool that helps us with formatting on commit, but I would say if we want to address it that we can address it when we come across it again.  It doesn't impact the tests.